src/resources/postcss/tickets-registration-page.pcss

```diff
@@ -186,7 +186,7 @@ body.page-tribe-attendee-registration {
 				-ms-grid-columns: 6fr 1fr 2fr 2fr;
 				grid-template-columns: 6fr 1fr 2fr 2fr;
 
-				&.tribe-tickets__item--price-suffix {
+				&.tribe-tickets__item--price-sufix {
```

Comment:
Not necessary since this is effectively the v2 version of the CSS, the v1 is a different stylesheet so it's still compatible

Reply:
Hey Scott, maybe there's a bit of confusion but the new version of the stylesheet[ lives now in ET+](https://github.com/moderntribe/event-tickets-plus/blob/bucket/ari-js-normalization/src/resources/postcss/tickets-registration-page.pcss). 

This revert, is part of the follow ups of the "Big PR". More precisely, [this comment](https://github.com/moderntribe/event-tickets/pull/1784#discussion_r490684192). So, it's basically reverting and leaving things as they were for the v1 of the AR page.